system/lib/compiler-rt/lib/sanitizer_common/sanitizer_flags.inc

```diff
@@ -71,7 +71,7 @@ COMMON_FLAG(bool, strip_env, 1,
             "avoid passing it to children. Default is true.")
 COMMON_FLAG(bool, detect_leaks, !SANITIZER_MAC, "Enable memory leak detection.")
 COMMON_FLAG(
-    bool, leak_check_at_exit, true,
+    bool, leak_check_at_exit, !SANITIZER_EMSCRIPTEN,
```

Comment:
Is this leak check thins part of the same change?

Reply:
It was part of this change, but it has now been superseded by commit 2d36b5e04587d18c142f78aa7e0e2dab539472ef.

Calling `__lsan_do_leak_check` by ourselves didn't quite work out for the `other.test_lsan*` tests.

Reply:
Commit https://github.com/emscripten-core/emscripten/commit/2d36b5e04587d18c142f78aa7e0e2dab539472ef didn't work for `asan.test_pthread_cxx_threads` somehow, it's superseded by commit https://github.com/emscripten-core/emscripten/pull/14481/commits/95e23faed1614e96e2a6f04e69728ef70617afdf now.

I should probably mark this PR as draft next time.

Reply:
The underlying issue will be resolved by PR #14625, I'm marking this conversation as resolved.